### ISSUE TITLE
Reproducing the results on the VOC dataset

### ISSUE THREAD

**Issue by kprokofi** [Alibaba-MIIL/ML_Decoder#14]
Hi, thanks for your work.
I tried to reproduce the results on the VOC dataset. I added the dataset to your code. This is [my fork](https://github.com/kprokofi/ML_Decoder).  I also added Timm wrapper to use timm models

I follow the article's recommendations and run the following command:
python train.py --data /path/to/voc --model-name  tresnet_l  --num-classes 20 --batch-size 48 --lr 2e-4

But the mAP isn't higher than 93.09% and the model overfits after 5 epochs. For the EfficientNetV2_small a similar picture is observed. The accuracy reaches 87% and the model overfits

Could you, please, help me to reproduce your results

**Comment by mrT23** [comment]
96.1% is not bad.

in any case, i trained with:
--learning_rate=5e-5 --batch_size=128

so i think you need lower learning rates.
play a bit larger batches, and reduced learning rates
